crud/insert.lua

```diff
@@ -147,7 +147,7 @@ function insert.object(space_name, obj, opts)
     opts = opts or {}
 
     -- insert can fail if router uses outdated schema to flatten object
-    opts.add_space_schema_hash = true
+    opts = utils.make_options(opts, true)
```

Comment:
Probably we don't need `opts = opts or {}`.

And make_options could be rewritten in following way:

```lua
if opts == nil then
    return {add_space_schema_hash  = add_space_schema_hash_value}
end

return {...}
```